account/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+
+
+class MyAccountManager(BaseUserManager):
+    def create_user(self, email, username, password=None):
+        if not email:
+            raise ValueError("Users must have an email address")
+        if not username:
+            raise ValueError("Users must have an username")
+
+        user = self.model(
+            email=self.normalize_email(email),
+            username=username,
+        )
+
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(self, email, username, password):
+        user = self.create_user(
+            email=self.normalize_email(email),
+            password=password,
+            username=username,
+        )
+        user.is_admin = True
+        user.is_staff = True
+        user.is_superuser = True
+        user.save(using=self._db)
+        return user
+
+
+class Account(AbstractBaseUser):
+    email = models.EmailField(verbose_name="email", max_length=60, unique=True)
+    username = models.CharField(max_length=30, unique=True)
+    date_joined = models.DateTimeField(verbose_name='date joined', auto_now_add=True)
+    last_login = models.DateTimeField(verbose_name='last login', auto_now=True)
+    is_admin = models.BooleanField(default=False)
+    is_active = models.BooleanField(default=True)
+    is_staff = models.BooleanField(default=False)
+    is_superuser = models.BooleanField(default=False)
```

Comment:
do we need all this is_admin, is_staff, is_superuser fields ? let's have an enum with 3 values, 
admin / read / write
something like this

Reply:
can you add `auto_login` field here as well `models.BooleanField(default=False)`

Reply:
> do we need all this is_admin, is_staff, is_superuser fields ? let's have an enum with 3 values,
> admin / read / write
> something like this

It is mandatory to have them because we use djzango.contrib.auth library, but anyways I included rights and auto_login columns in the model

Reply:
> can you add `auto_login` field here as well `models.BooleanField(default=False)`

Done, Sir

Reply:
LGTM